boards/arm/nucleo_l011k4/doc/index.rst

```diff
@@ -0,0 +1,170 @@
+.. _nucleo_l011k4_board:
+
+ST Nucleo L011K4
+################
+
+Overview
+********
+The STM32 Nucleo-32 development board with STM32L011K4 MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board provides an affordable, and flexible way for users to try out new concepts,
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption, and features.
+
+The Arduino* Uno V3 connectivity support and the ST morpho headers allow easy functionality
```

Comment:
To be upated

Reply:
Updated to remove ST morpho and change Arduino Uno to Arduino Nano